rj_gameplay/rj_gameplay/tactic/line_tactic.py

```diff
@@ -1,22 +1,24 @@
 import stp
 from rj_gameplay.role import dumb_move
 from typing import List, Tuple
-
 from rj_msgs.msg import RobotIntent
 
 
 class LineTactic(stp.tactic.Tactic):
     """Tactic for line up play that puts all six robots in a line on the left of the field."""
 
-    def __init__(self, world_state: stp.rc.WorldState):
+    def __init__(self, world_state: stp.rc.WorldState, num_lined_up: int):
         super().__init__(world_state)
 
         # compute move points
-        # TODO: make start on side of the field, so this Tactic is actually useful during penalty situations
-        start = (3.0, 0.0)
+        # access the length and the width of the field (plus the border), and starts robots there
+        start = (
+            (world_state.field.width_m + world_state.field.border_m) / 2,
+            (world_state.field.length_m - world_state.field.border_m) / 2,
+        )
```

Comment:
Again don't know if these set of points are legal for a penalty kick, maybe make a separate tactic for penalties, or allow this one to be adjustable somehow.